actions/get-charm-paths/src/get_charm_paths.py

```diff
@@ -3,59 +3,86 @@
 import os
 from pathlib import Path
 
-
 OUTPUT_VARIABLE_NAME = "charm_paths"
 
+
 def find_charms_in_dir(base_dir: str = "./", charms_subdir="charms"):
     """
     Finds paths to charm directories in base_dir.
 
-    Directories are identified as charm directories by the presence of a "metadata.yaml" or "metadata.yml" file.
+    Directories are identified as charm directories by the presence of a
+    "metadata.yaml", "metadata.yml" or "charmcraft.yaml" file.
 
     Returns:
-    * If any subdirectories of base_dir/charms_subdir/ are charm directories, then the list of these directories is returned.
-    * If not, then if the base directory itself is a charm directory, that is returned.
+    * If any subdirectories of base_dir/charms_subdir/ are charm directories,
+      then the list of these directories is returned.
+    * If not, then if the base directory itself is a charm directory, that is
+      returned.
     * Otherwise, returns []
 
-	Directories returned are alphabetically sorted.
+        Directories returned are alphabetically sorted.
     """
-    base_dir = Path(base_dir)
-    charms_dir = base_dir / charms_subdir
-    
-    metadata_files = list(charms_dir.glob("*/metadata.yaml")) + list(charms_dir.glob("*/metadata.yml"))
-    
+    root_dir = Path(base_dir)
+    charms_dir = root_dir / charms_subdir
+
+    metadata_files = (
```

Comment:
```suggestion
    tracked_files = (
```
? 
or sth like that

Reply:
Sent a commit 322e2be